src/cookbooks/new_ping.md

```diff
@@ -185,10 +185,13 @@ a [Longitudinal](https://github.com/mozilla/telemetry-batch-view/blob/master/src
 or [client-count-daily](https://github.com/mozilla/telemetry-batch-view/blob/master/src/main/scala/com/mozilla/telemetry/views/GenericCountView.scala)
 like dataset. Otherwise, you'll have to write your own.
 
-You can schedule it on [Airflow](http://workflow.telemetry.mozilla.org/), or you can
+You can schedule it on [Airflow], or you can
 run it as a job in ATMO. If the output is parquet, you can add it to the Hive metastore to have it
 available in re:dash. Check the docs on [creating your own datasets](create_a_dataset.md).
 
 Build Dashboards Using ATMO or STMO
 -----------------------------------
 Last steps! What are you using this data for anyway?
+
+
+[Airflow]: https://r3e51de5feeeb5619-tp.appspot.com/
```

Comment:
Let's use the stable `mozaws` url mentioned in the bug for now.
```suggestion
[Airflow]:  https://workflow.prod.mozaws.net
```

Reply:
Does that link work for you? I think it has the same challenge as w.t.m.o would (i.e. something would have to host a http redirect).

Reply:
Argh, nope. Merging the PR as-is.